Regression/WarpX-tests.ini

```diff
@@ -473,6 +473,25 @@ analysisRoutine = Examples/Tests/Langmuir/analysis_langmuir_multi_2d.py
 analysisOutputImage = langmuir_multi_2d_analysis.png
 tolerance = 1.e-14
 
+[Langmuir_multi_2d_MR]
+buildDir = .
+inputFile = Examples/Tests/Langmuir/inputs_2d_multi_rt
+runtime_params = algo.maxwell_solver = ckc  warpx.use_filter = 1  amr.max_level = 1  amr.ref_ratio = 4  warpx.fine_tag_lo = -10.e-6 -10.e-6  warpx.fine_tag_hi = 10.e-6 10.e-6  diag1.electrons.variables = w ux uy uz  diag1.positrons.variables = w ux uy uz
```

Comment:
Here is where we set the input parameters relevant for mesh refinement: `amr.max_level`, `amr.ref_ratio`, `warpx.fine_tag_lo` and `warpx.fine_tag_hi`.